plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java

```diff
@@ -65,6 +66,7 @@ private PDU createPDU(SnmpTrapInfo snmpTrapInfo) {
 
         int alertType = snmpTrapInfo.getAlertType() + 1;
         if (alertType > 0) {
+            trap.add(new VariableBinding(SnmpConstants.sysUpTime, new OctetString(new Date().toString())));
```

Comment:
This doesn't seem what we want. The sysUpTime shouldn't be set to 'now', should it? Without to much diving in I'd say it should be added and retrieved from the SnmpTrapInfo. If I am wrong, please add a comment explaining why 'now' is alright.